Documentation/conf.py

```diff
@@ -132,3 +134,16 @@
 # -- Options for warnings_filter ------------------------------------------
 
 warnings_filter_config = "known-warnings.txt"
+
+# -- Options for sphinx_tags ----------------------------------------------
+
+tags_create_tags = True
```

Comment:
I see that `Tags` page appears at the top level menu. Shouldn't this appear in the "Supported Platforms" menu if we want to track boards features?

Reply:
I thought it would be best to give it its own page, since I plan for tags to eventually cover more than just boards. For instance, device drivers can be marked as "experimental" as well right now.

Reply:
yup +1 for the Tags in the top menu as proposed for quick search :-)